pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -87,9 +87,8 @@ func getKey(ds *extensions.DaemonSet, t *testing.T) string {
 	if key, err := controller.KeyFunc(ds); err != nil {
 		t.Errorf("Unexpected error getting key for ds %v: %v", ds.Name, err)
 		return ""
-	} else {
-		return key
 	}
+	return key
```

Comment:
key needs to be initialized above the if clause.

Reply:
Good point, thanks.